From c60d9dd109978cb24572644fcf1317c9fef52dd9 Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Mon, 23 Dec 2019 12:20:39 +0200
Subject: [PATCH 10/16] Move custom DPI settings in to Display menu

  * from
  https://github.com/Havoc-OS/android_packages_apps_Settings/commit/ae0244c957ec115822da526d54ee3665488e50ef

	modified:   res/xml/development_settings.xml
	modified:   res/xml/display_settings.xml
	modified:   src/com/android/settings/development/DevelopmentSettingsDashboardFragment.java
---
 res/xml/development_settings.xml                              | 4 ----
 res/xml/display_settings.xml                                  | 4 ++++
 .../development/DevelopmentSettingsDashboardFragment.java     | 1 -
 3 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/res/xml/development_settings.xml b/res/xml/development_settings.xml
index 7a8ddeced2..7d01f6924d 100644
--- a/res/xml/development_settings.xml
+++ b/res/xml/development_settings.xml
@@ -449,10 +449,6 @@
             android:entries="@array/overlay_display_devices_entries"
             android:entryValues="@array/overlay_display_devices_values" />
 
-        <com.android.settings.display.DensityPreference
-            android:key="density"
-            android:title="@string/developer_smallest_width" />
-
         <ListPreference
             android:key="display_cutout_emulation"
             android:title="@string/display_cutout_emulation"
diff --git a/res/xml/display_settings.xml b/res/xml/display_settings.xml
index 9bd37f4a22..e27a126d3e 100644
--- a/res/xml/display_settings.xml
+++ b/res/xml/display_settings.xml
@@ -121,6 +121,10 @@
         android:entries="@array/refresh_rate_entries"
         android:entryValues="@array/refresh_rate_values" />
 
+    <com.android.settings.display.DensityPreference
+        android:key="density"
+        android:title="@string/developer_smallest_width" />
+
     <SwitchPreference
         android:key="show_operator_name"
         android:title="@string/show_operator_name_title"
diff --git a/src/com/android/settings/development/DevelopmentSettingsDashboardFragment.java b/src/com/android/settings/development/DevelopmentSettingsDashboardFragment.java
index 4b7a765ae0..7f2c31fd57 100644
--- a/src/com/android/settings/development/DevelopmentSettingsDashboardFragment.java
+++ b/src/com/android/settings/development/DevelopmentSettingsDashboardFragment.java
@@ -522,7 +522,6 @@ public class DevelopmentSettingsDashboardFragment extends RestrictedDashboardFra
         controllers.add(new DefaultLaunchPreferenceController(context, "feature_flags_dashboard"));
         controllers.add(
             new DefaultLaunchPreferenceController(context, "default_usb_configuration"));
-        controllers.add(new DefaultLaunchPreferenceController(context, "density"));
         controllers.add(new DefaultLaunchPreferenceController(context, "background_check"));
         controllers.add(new DefaultLaunchPreferenceController(context, "inactive_apps"));
         controllers.add(new AutofillLoggingLevelPreferenceController(context, lifecycle));
-- 
2.25.1

